lib/blocs/area/cityListBloc/CityListEvent.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:equatable/equatable.dart';
+
+abstract class CityListEvent extends Equatable {
+  const CityListEvent([List props = const <dynamic>[]]);
```

Comment:
Avoid defining unused parameters in constructors.
`[List props = const <dynamic>[]]` hapus aja, tambahin override props di body class abstract-nya. Jadi gini:
```
abstract class CityListEvent extends Equatable {
   const CityListEvent();
   
   @override
   List<Object> get props => <Object>[];
}
```

Supaya nanti kelas2 turunanannya yang ga punya variabel ga perlu implementasi override props lagi.